pkg/cmd/miscellaneous.go

```diff
@@ -327,22 +327,8 @@ func capture() func() (string, error) {
 	}
 }
 
-func isIP(word string) bool {
-	parts := strings.Split(word, ".")
-	if len(parts) < 4 {
-		return false
-	}
-	for _, x := range parts {
-		if i, err := strconv.Atoi(x); err == nil {
-			if i < 0 || i > 255 {
-				return false
-			}
-		} else {
-			return false
-		}
-
-	}
-	return true
+func isIP(host string) bool {
+	return net.ParseIP(host) != nil
 }
```

Comment:
Even if the function was named isIP, the old implementation would only return true in case it was an ipv4 address.
Now it will return true if it's an ipv6 and ipv4 address..

Reply:
should I rename isIP to like isIPv4 something like that then check and return only ipv4 address ?

Reply:
yes I guess this would be better. It's only used in one place currently `createBastionHostInstance` (gcp). Not sure if this will work with ipv6 addresses..

Reply:
fixed, so far GCP `createBastionHostInstance`  not support ipv6 yet I think, but I will check it later when I work on support GCP host in the `user` role later.